packages/medusa/src/api/routes/admin/products/index.ts

```diff
@@ -258,6 +260,8 @@ export type AdminProductsDeleteRes = {
  *     - variants
  *     - variants.options
  *     - variants.prices
+ *   totals:
```

Comment:
question: Why did we add it under `totals`?

Reply:
Talked with @patrick-medusajs, it's a custom prop that is just used for decorated properties, It's just a heading to let oas know that there's an additional prop being populated. We could also add a new oas prop that's called `decorated` or something similar to that, I give you that `totals` is not a good name in this case 😄

Reply:
After our conversation, I will probably rename `totals` to something like `calculated`, `computed`, `dynamic`. Or I might keep `totals` and introduce a new term. I'm opened for suggestions :)